.github/workflows/shared-issues.yml

```diff
@@ -1,4 +1,6 @@
-name: "Shared Issues"
+# Runs automation internal to airbyte for Issues
```

Comment:
That's because it's really super generic. If you look at the repo it points to, it's actually doing several different things and this was the way I could capture all of them. I do like your description a bit better though